Token.js

```diff
@@ -717,7 +681,7 @@ class Token {
 }
 
 
-function token_button(e) {
+function token_button(e, tokenIndex = null, tokenTotal = null) {

```

Comment:
I would set these to empty arrays, then check the length on the array rather than expecting null as a valid primitive. Usually leads to issues/bugs.

Reply:
Correction, since these are Number prims of an index, I would suggest setting them to 0 then using Number.isNumber(var). 

Good rule of thumb is never accept null or mix your privatives loosely typed languages.

Reply:
that would invalidate the reason why we use it (create a circular placement of tokens when using the add party button), 0 is a valid index and in this case a valid total.